qemu/tests/balloon_check.py

```diff
@@ -91,6 +92,54 @@ def memory_check(self, step, ballooned_mem):
                 raise exceptions.TestFail("Balloon test failed %s" % step)
         return (mmem, gmem)
 
+    def memory_stats_check(self, keyname):
+        """
+        Check memory statistics via balloon service.
+
+        :param keyname: key name of the output of the 'qom-get' property.
+        :return: if memory statistics can be got via balloon service, reture True,
+                 else return False.
+        :rtype: bool
+        """
+        check_mem_ratio = float(self.params.get("check_mem_ratio", 0.1))
+        sleep_time = int(self.params.get("polling_sleep_time", 20))
+        base_path = self.params.get("base_path", "/machine/peripheral/")
+        device = self.params.get("balloon", "balloon0")
+        device_path = os.path.join(base_path, device)
```

Comment:
The `qom` path is not the same as filesystem path, you can just join these two string into one by `+`.

Reply:
Updated, thanks